### ISSUE TITLE
Create charting on connector.land for connector fees over time #413

### ISSUE THREAD

**Issue by ghost** [michielbdejong/connector.land#4]
Display charts of price of connector fees over time, average settlement time and other statistics.

**Comment by michielbdejong** [comment]
Yes, I was in the process of adding that (now, the #ledgers tab only shows reachable or not), and that was how I found out about the whole routing vs quoting thing, and that's kept me busy since then. Next week, we'll change the default setting for ilp-kit to obey curves, and once that's done, I'll:

* [ ] publish each destination's combined curve on the #ledgers tab.
* [ ] do test payments based on curves, and report success rate and delay.
